### PR TITLE
migrate to gradle 6

### PR DESCRIPTION
**Pull request by operasfantom**

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.0.1-all.zip
```
Comment (ForNeVeR) [inline gradle/wrapper/gradle-wrapper.properties]: Could you please execute `./gradlew wrapper` to regenerate the stuff in the [`gradle/wrapper` directory](https://github.com/JetBrains/rd/tree/master/gradle/wrapper), too?

Comment (operasfantom) [reply gradle/wrapper/gradle-wrapper.properties]: How could I forget about that? Actually it's relevant at the moment

File: buildSrc/src/main/kotlin/com/jetbrains/rd/gradle/tasks/MarkedExecTask.kt [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
 
 /**
  * Provides system specific command line. It never starts directly or not.
  */
 interface MarkedExecTask/* : Task*/ {
+    @get:Input
     val commandLineWithArgs: List<String>
 
+    @Internal
     fun getWorkingDir(): File
```
[the remark this answers is not in the document]
Comment (operasfantom) [reply buildSrc/src/main/kotlin/com/jetbrains/rd/gradle/tasks/MarkedExecTask.kt]: This method binds to appropriate method in _org.gradle.api.Task_ via derived class, where it's annotated as  **@Internal**. Can not be marked differently due to compile errors.

**Comment by operasfantom** [comment]
Can we merge it now?

**Pull request merged**